src/vfd/sriov.c

```diff
@@ -267,6 +267,10 @@ set_vf_rx_mac(portid_t port_id, const char* mac, uint32_t vf,  __attribute__((__
   struct ether_addr mac_addr;
   ether_aton_r(mac, &mac_addr);
 
+	if (mac_addr.addr_bytes[0] & 0x1) {
```

Comment:
You should be able to use is_valid_assigned_ether_addr() instead (defined in rte_ether.h).  That will also protect against a zero MAC address.